cdap-docs/developers-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -317,6 +317,58 @@ For example, give these arguments when starting the MapReduce through a RESTful
     "dataset.totals.output.partition.key.league" : "nfl"
   }
 
+Incrementally Processing Partitions of a PartitionedFileSets in MapReduce
+=========================================================================
+
+One way to process a partitioned file set is with a MapReduce program that runs repeatedly and,
```

Comment:
One way to process a partitioned fileset is with a repeatedly-running MapReduce program that,